Dockerfile

```diff
@@ -13,8 +14,13 @@ apt-get install --no-install-recommends -y \
 unzip \
 ca-certificates \
 wget && \
+apt-get install -y sqlite3 libsqlite3-dev && \
 apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+COPY bin ${SCRIPT_SRC}/bin
+RUN chmod +x ${SCRIPT_SRC}/bin/*
+RUN ln -snf ${SCRIPT_SRC}/bin/* /usr/local/bin
```

Comment:
@bjb28 Is there a reason you had in mind for not just copying `bin/get-api` directly into `/usr/local/bin`?  That seems like a simpler way to go here.  All you would need to do is:
```
SCRIPT_DIR="/usr/local/bin"

COPY bin/get-api {SCRIPT_DIR}
RUN chmod +x {SCRIPT_DIR}/get-api
```

@felddy and @jsf9k - feel free to weigh in.  This will be a handy script to have in our gophish container.

Reply:
Agreed, @dav3r's solution seems much simpler.

Reply:
Changes made and pushed.